app/uk/gov/hmrc/helptosavefrontend/views/reminder/reminder_confirmation_update.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.helptosavefrontend.views.html.helpers._
+@import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+@import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavefrontend.controllers.routes
+
+
+@this(main: uk.gov.hmrc.helptosavefrontend.views.html.main_template,button: uk.gov.hmrc.helptosavefrontend.views.html.helpers.button)
+
+@(email: String,period :String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+
+@title = @{
+        Messages("hts.reminder-confirmation_update.title.h1")
+     }
+
```

Comment:
You can avoid this whole file of duplication, by something like:

@(email: String,period :String, headMessage : String, paraMessage : String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
 
  @title = @{
          Messages(hadingMessage)
       }

...

The just call from uk.gov.hmrc.helptosavefrontend.controllers.ReminderController#getRendersConfirmPage with something like:

          Ok(if (page === "Set") {reminderConfirmation(value, period"hts.reminder-confirmation-set.title.h1", "hts.reminder-confirmation_update.title.p1"))
          } else {reminderConfirmation(value, period, "hts.reminder-confirmation-set.title.h1", "hts.reminder-confirmation_update.title.p1"))
          }

Better DRY. Do you see?